Documentation/network/servicemesh/http.rst

```diff
@@ -92,3 +92,51 @@ get back some data, but you will get a 404 error if you make a request to ``<add
       "ISBN-10": "1234567890",
       "ISBN-13": "123-1234567890"
     }
+
+Apply Cilium Network Policy
+===========================
+
+The above example doesn't include any network policy, so all the external traffic is allowed by default.
```

Comment:
"Above" in this context sometimes trips up translators. Can we change this to 'previous'?

```suggestion
The previous example doesn't include any network policy, so all the external traffic is allowed by default.
```